src/ui/posting/Entry.css

```diff
@@ -27,3 +27,21 @@
 .entry .content P IMG {
     max-width: min(900px - 3.5rem, 100vw - 4rem);
 }
+
+details.spoiler {
```

Comment:
Please, write tag names in caps: `DETAILS.spoiler`.

Reply:
And better to limit the scope of the style to `.entry` or even `.entry .content`, like it is done in styles above.